Tests/AsyncHTTPClientTests/HTTPClientInternalTests.swift

```diff
@@ -797,7 +797,7 @@ class HTTPClientInternalTests: XCTestCase {
     }
 
     func testUncleanCloseThrows() {
-        let server = NIOHTTP1TestServer(group: self.clientGroup)
+        let server = NIOHTTP1TestServer(group: MultiThreadedEventLoopGroup(numberOfThreads: 1))
```

Comment:
who shuts this MTELG down?

Reply:
I think we have `self.serverGroup` which could be used

Reply:
oooops, no one, good catch, thank you! no, we don't have that in internal tests class

Reply:
and its only monday... 😔

Reply:
> oooops, no one, good catch, thank you! no, we don't have that in internal tests class

We should. I was probably lazy and changed only all of the non-internal tests

Reply:
so far we won't be using it anywhere else in that class (it usually just creates `HTTPBin`), still worth it?

Reply:
@artemredkin I think we should have a `defaultHTTPBin` and `defaultClient` and `client/serverGroup` all set up in `setUp`/`tearDown`. That makes tests more concise and therefore better

Reply:
but we don't need to do it right in this PR, up to you